Core/Product Request/PaymentTransactionObserver.swift

```diff
@@ -79,19 +79,23 @@ public class PaymentTransactionObserver: NSObject, SKPaymentTransactionObserver
     }
     
     public func paymentQueueRestoreCompletedTransactionsFinished(_ queue: SKPaymentQueue) {
-        // Check to see if the returned queue as any transactions.  If there are no
-        // transactions, it means the user has not made any purchases yet, so there is
-        // nothing to restore.
-        guard queue.transactions.count > 0 else {
-            // There were no transactions for this user
-            let localizedErrorMessage = NSLocalizedString("Sorry, no purchased products were found. If you have questions, please contact support@blackpixel.com.", tableName: nil, bundle: Bundle(for: PaymentTransactionObserver.self), value: "", comment: "no previous transactions were found on the users current iTunes account error message")
-            
-            delegate?.transactionDidFail(localizedErrorMessage: localizedErrorMessage)
+        // fetch support email or nil if it doesn't exist
+        if let info = Bundle.main.infoDictionary,
+           let supportEmail = info["HGSupportEmail"] as? String {
```

Comment:
What happens if there is no support email defined? I think an error message should still be presented to let the user know something went wrong.

Reply:
No purchases are downloaded.  This observer is not necessarily on the main thread and it does not have a viewcontroller to easily present an alert.  The other alert is already presented on the application side, I'll put one over there if we detect purchases